OpenRA.Mods.Common/Widgets/Logic/MainMenuLogic.cs

```diff
@@ -32,6 +36,22 @@ protected enum MenuType { Main, Singleplayer, Extras, MapEditor, None }
 		// Update news once per game launch
 		static bool fetchedNews;
 
+		// Increment the version number when adding new stats
+		const int SystemInformationVersion = 1;
+		Dictionary<string, Pair<string, string>> GetSystemInformation()
+		{
+			var lang = System.Globalization.CultureInfo.InstalledUICulture.TwoLetterISOLanguageName;
+			return new Dictionary<string, Pair<string, string>>()
+			{
+				{ "id", Pair.New("Anonymous ID", Game.Settings.Debug.UUID) },
+				{ "platform", Pair.New("OS Type", Platform.CurrentPlatform.ToString()) },
+				{ "os", Pair.New("OS Version", Environment.OSVersion.ToString()) },
+				{ "runtime", Pair.New(".NET Runtime", Platform.RuntimeVersion) },
```

Comment:
The .NET Runtime used on windows is the .NET CLR 4.0.x.  There's no redundancy here.